chat-plugins/info.js

```diff
@@ -109,6 +109,19 @@ var commands = exports.commands = {
 		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
 			buf += '<br />Private rooms: ' + privaterooms;
 		}
+
+		if (user.can('lock') || (user.can('roomban') && Object.keys(room.users).indexOf(targetUser.userid) >= 0)) {
```

Comment:
I don't remember if this works entirely, but I think if you want to check for room modship and check if, say, your fellow mods have been room banned, I think `user.can('ban', null, room)` might work.